.github/ISSUE_TEMPLATE/vsp-release-plan.md

```diff
@@ -0,0 +1,86 @@
+# Release Plan Template [Draft]
```

Comment:
this needs a stanza at the top with template defaults

```
---
name: VSP Initiative Release Plan Template (Draft)
about: [... some description here ...]
title: "[Initiative-Name]: Release Plan"
---
```

and are there any labels we want by default?